neo/Storage/Interface/DBInterface.py

```diff
@@ -2,11 +2,35 @@
 from neo.SmartContract.Iterable import EnumeratorBase
 from neo.logging import log_manager
 
-logger = log_manager.getLogger('db')
+logger = log_manager.getLogger()
 
 
-class DBCollection:
-    DB = None
+class DBProperties:
+    """
+    Container for holding DB properties
+    Used to pass the configuration options to the DB iterator initializer.
+    neo.Storage.Implementation.[BACKEND].[BACKEND]Impl.openIter
+
+    Args:
+        prefix (str, optional): Prefix to search for.
+        include_value (bool, optional): whether to include keys in the returned data
+        include_key (bool, optional): whether to include values in the returned data
+    """
+
+    prefix = None
+    include_value = None
+    include_key = None
```

Comment:
Please remove the class variables. I think they should just be instance variables

Reply:
Done.